design_drafts/components_architecture_and_urdf_examples.md

```diff
@@ -171,7 +170,12 @@ Note:
       <state_interface name="position"/>
     </joint>
     <sensor name="tcp_fts_sensor">
-      <classType>ros2_control_components/ForceTorqueSensor</classType>
```

Comment:
If we remove this, how should we then load specific Sensor component.

Very concrete example from ROS1:

Package  KITrobotics/force_torque_sensor implements FTS abstraction with logical code reused by different FTS-Sensors.  KITrobotics/ati_force_torque implements CAN and RS485 drivers for the ATI-FTS hardware. (The package for other sensor Schuk FTC50 is currently not public)

Accessing `force_torque_sensor` abstraction from the hardware makes would make implementations unnecessary complicated. This means that every HW-package would need to implement the node or something similar.
Therefore, I prefer that we provide an option to load complex Sensors and Joints (an example for this should be added)

Reply:
the hardware component is being loaded in the hardware tag, not the sensor tag.